savoirfairelinux/num2words
# add Iranian number to word

## Fixes # by...

### Changes proposed in this pull request:

* .add lang_fa


### Status


### How to verify this change
use test file.
*Fill out this section so that a reviewer can know how to verify your change.*

### Additional notes

*If applicable, explain the rationale behind your change.*
 add Iranian(Farsi) number to word
